Microsoft.Toolkit.Uwp.UI.Controls/ImageCropper/ImageCropperCroppedRegionChangedEventArgs.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.Foundation;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    ///为< 请参阅cref = “ ImageCropper.CroppedRegionChanged ” />事件提供事件数据。 
```

Comment:
Looks like something happened with GitHub accepting the comment? This is causing the CI to fail as it's not a valid XML doc comment anymore.

Reply:
It seems that github accepted the changes that were automatically translated by Google Translate (because my chrome automatically translated the webpage into Chinese), maybe it should be a github bug? 🤣
 Let me fix it.